deploy/charts/cert-manager/values.yaml

```diff
@@ -206,6 +206,14 @@ webhook:
   replicaCount: 1
   timeoutSeconds: 10
 
+  # Used to configure options for the webhook pod.
+  # This allows setting options that'd usually be provided via flags.
+  # An APIVersion and Kind must be specified in your values.yaml file.
+  # Flags will override options that are set here.
+  config:
```

Comment:
Thought: may have been nice if there was some way of validating the fields, at the moment the webhook pod will just crash if an invalid field was added.
Also because webhook does not reload the config by default, it may happen that someone applies a change with some invalid fields and it just sits in the configmap for a while and the deployment fails at some point later.
However, I guess it would not be straightforward since any validation would need to be for a specific api version..

Reply:
Yep, we don't have any good way to reliably validate this (especially not in a backward compatible way) - we'd need a `v1alpha1` field, `v1alpha2` etc etc (and would need to maintain the Helm schema file/generate it from our own resource types).

At least for now, I think this is something to punt to further down the line (especially as we may have ways to configure via the apiserver in the future too anyway)

Reply:
Agree 👍🏼